src/components/dashboard/SalesByDayOfWeekChart.tsx

```
@@ -0,0 +1,171 @@
+        fill={isDark ? '#F3F4F6' : '#374151'}
+        textAnchor='middle'
+        fontSize='12'
+        fontWeight='500'
+      >
+        {`${percentage.toFixed(1)}%`}
+      </text>
+    )
+  }
+
+  // Custom tooltip
+  function CustomTooltip({ active, payload, label }: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)